src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -74,6 +74,56 @@ public Size(int width, int height)
         public static Size operator -(Size sz1, Size sz2) => Subtract(sz1, sz2);
 
         /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="multiplier">Multiplier of type <see cref="int"/>.</param>
+        /// <param name="size">Multiplicand of type <see cref="Size"/>.</param>
+        /// <returns>Product of type <see cref="Size"/>.</returns>
+        public static Size operator *(int multiplier, Size size) => Multiply(size, multiplier);
+
+        /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="size">Multiplicand of type <see cref="Size"/>.</param>
+        /// <param name="multiplier">Multiplier of type <see cref="int"/>.</param>
+        /// <returns>Product of type <see cref="Size"/>.</returns>
+        public static Size operator *(Size size, int multiplier) => Multiply(size, multiplier);
+
+        /// <summary>
+        /// Division operator that divides <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="size">Dividend of type <see cref="Size"/>.</param>
+        /// <param name="divisor">Divisor of type <see cref="int"/>.</param>
+        /// <returns>Result of type <see cref="Size"/>.</returns>
+        public static Size operator /(Size size, int divisor) =>
```

Comment:
nit: We should make this do multiplication by reciprocal, as we do in the System.Numerics.Vector* types.